autogluon/inference/docker/0.6/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,65 @@
+ARG PYTHON_VERSION=3.8.10
+
+FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/mxnet-inference:1.9.0-cpu-py38-ubuntu20.04-sagemaker
+
+# PYTORCH_SKIP_CUDNN_COMPATIBILITY_CHECK - see https://github.com/autogluon/autogluon/issues/2534
+ENV MXNET_CUDNN_LIB_CHECKING=0 \
+    MXNET_CUDNN_AUTOTUNE_DEFAULT=0 \
+    PYTORCH_SKIP_CUDNN_COMPATIBILITY_CHECK=1
+
+# Specify accept-bind-to-port LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
+# https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+# Specify multi-models LABEL to indicate container is capable of loading and serving multiple models concurrently
+# https://docs.aws.amazon.com/sagemaker/latest/dg/build-multi-model-build-container.html
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+RUN apt-get update \
+ && apt-get -y upgrade \
+ && apt-get autoremove -y \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+ARG AUTOGLUON_VERSION=0.6.1
+ARG TORCH_VER=1.12.1+cpu
+ARG TORCH_VISION_VER=0.13.1+cpu
+ARG TORCH_TEXT_VER=0.13.1
+
+# NOTE: reinstalling horovod with pytorch support because it is available in the container
+# Updating certifi - https://pyup.io/p/pypi/certifi/2ff/
+RUN pip install --no-cache-dir -U --trusted-host pypi.org --trusted-host files.pythonhosted.org pip \
+ && pip install --no-cache-dir -U setuptools wheel \
+ && pip uninstall -y dataclasses \
+ && pip install --no-cache-dir -U certifi \
+ && pip install --no-cache-dir -U numpy numba \
+    torch==${TORCH_VER} torchvision==${TORCH_VISION_VER} torchtext==${TORCH_TEXT_VER} --extra-index-url https://download.pytorch.org/whl/cpu \
+ && pip install --no-cache-dir -U autogluon==${AUTOGLUON_VERSION} \
+ && mim install -q mmcv-full \
+ && pip install --no-cache-dir -U mmdet
+
+# Downgrade protobuf 3.20.1 -> 3.19.6: https://pyup.io/v/51167/f17
+RUN pip install -U protobuf==3.19.*
```

Comment:
Same here.

This downgrade seems to be unnecessary to v0.6.1, since we've upgrade torch==1.11.0. See https://github.com/autogluon/autogluon/pull/2193

Reply:
ditto - see above